### ISSUE TITLE
> you need to subscribe on this function `func willSelectDate(_ date: Date, type: CalendarType) { print(date, type) calendarView.reloadData() }`

### ISSUE THREAD

**Issue by dhavalSA** [kvyatkovskys/KVKCalendar#364]
> you need to subscribe on this function `func willSelectDate(_ date: Date, type: CalendarType) { print(date, type) calendarView.reloadData() }`

@kvyatkovskys this solution works only when we have style.month.autoSelectionDateWhenScrolling = true, in my case, I am using style.month.autoSelectionDateWhenScrolling = false to have the same user experience as Apple's default calendar, when we set style.month.autoSelectionDateWhenScrolling = false willSelectDate is not getting called then how can I show events without selecting any date

_Originally posted by @dhavalSA in https://github.com/kvyatkovskys/KVKCalendar/issues/241#issuecomment-2208654969_

**Comment by dhavalSA** [comment]
@kvyatkovskys is it possible to guide me here, on how I can fix it, I need this working,  I need only guidance I will fix it on my own if you are busy

**Comment by kvyatkovskys** [comment]
Hi, I'll take a look and give you an answer today

**Comment by kvyatkovskys** [comment]
I checked code and it's impossible right now... Can you check this branch `feature/v_0.6.25` I added some changes

**Comment by dhavalSA** [comment]
@kvyatkovskys thanks , i will check and let you know

**Comment by dhavalSA** [comment]
@kvyatkovskys still same issue

**Comment by kvyatkovskys** [comment]
Can you describe what do you need? Cos for me all works fine

**Comment by dhavalSA** [comment]
@kvyatkovskys  assume that I have 3 events in July and 3 events in August, I am using Month view for now, when I load the calendar with today's date selected, it shows 3 events in July month, now if I am scrolling to August month where I also have 3 events but these events are not visible on the calendar until I select any date on August month, in short, we can say, for any month we need t select any date on that month to make events visible on the calendar  

FYI: in my calendar setup I have style.month.autoSelectionDateWhenScrolling = false as it's desired behaviour I want

**Comment by plankton4** [comment]
@dhavalSA Hi! Were you able to solve this problem?

**Comment by dhavalSA** [comment]
@plankton4 yes, I forked a copy and did some development to add a few more features, due to some important release, I was not able to wait for a new version. Also, there are a few more requirements that were missing, so I preferred this  way
